input/templates/partials/common/wishlist.hbs

```diff
@@ -9,7 +9,9 @@
         <span class="recently-added-title">{{i18n "catalog:wishlist.recentlyAdded"}}</span>
         <div class="clearfix">
             {{#each wishlist.list}}
-                <img class="pull-left img-responsive wishlist-img" src="{{imageUrl}}" alt="{{name}}">
+                <a href="{{productSlug}}">
```

Comment:
This shouldn't be called `productSlug` because it won't be a slug, in the end it will be a URI in any case. I know in our particular implementation in Sunrise Java app we use the slug to generate the URI, but the theme should not know anything about it. So to follow the naming in cart, I would just call it `url` for now. The names and structure need to be reworked in the future anyway.